system/lib/fetch/emscripten_fetch.cpp

```diff
@@ -125,27 +169,44 @@ EMSCRIPTEN_RESULT emscripten_fetch_wait(emscripten_fetch_t *fetch, double timeou
 #endif
 }
 
-EMSCRIPTEN_RESULT emscripten_fetch_close(emscripten_fetch_t *fetch)
+EMSCRIPTEN_RESULT emscripten_fetch_close( emscripten_fetch_t *fetch )
 {
-	if (!fetch) return EMSCRIPTEN_RESULT_SUCCESS; // Closing null pointer is ok, same as with free().
+    if ( !fetch ) return EMSCRIPTEN_RESULT_SUCCESS; // Closing null pointer is ok, same as with free().
```

Comment:
Here and throughout, it looks like the code is going away from convention, i.e. from `(!fetch)` to `( !fetch )`, but ideally would be the opposite. Also would be good to stick to tabs for now (follow the convention of surrounding code (tm)). And let's do the style changes in a followup PR.